dali/test/python/test_pipeline.py

```diff
@@ -1707,3 +1707,14 @@ def get_output():
     out = get_output()[0].at(0)
     assert out[0] == -0.5 and out[1] == 1.25
 
+def test_return_constants():
+    pipe = dali.pipeline.Pipeline(1, 1, None)
+    types = [bool, np.int8, np.uint8, np.int16, np.uint16, np.int32, np.uint32, np.float32]
```

Comment:
NOTE: Constant operator doesn't support int64 or double at native level. We should probably fix it at some point somehow, but I don't have a good idea of how to do that at present.